runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBrowseView/Header.tsx

```diff
@@ -57,18 +57,20 @@ const Header: FC<HeaderProps> = ({ className, ...rest }) => {
 
     const onTabChange = (event: ChangeEvent<HTMLInputElement>, value: ProcessesTabs) => {
         if (value === ProcessesTabs.COLLECTIONS) {
-            router.replace({ 
+            router.replace({
                 pathname:  '/app/processes/collections',
                 query: {
-                    collection: collectionId,
-                    pageSize: pageSize, 
+                    collectionId,
+                    locale: router.locale,
```

Comment:
as far as I'm concerned we don't need to pass the locale - the useTranslations hook takes care of that